vzstorage-pd/virtuozzo-provisioner.go

```diff
@@ -41,18 +40,11 @@ import (
 )
 
 const (
-	resyncPeriod              = 15 * time.Second
-	provisionerName           = "kubernetes.io/virtuozzo-storage"
-	exponentialBackOffOnError = false
-	failedRetryThreshold      = 5
-	parentProvisionerAnn      = "vzFSParentProvisioner"
-	vzShareAnn                = "vzShare"
+	provisionerName      = "kubernetes.io/virtuozzo-storage"
```

Comment:
I suggest to use virtuozzo.io as it is more fashionable nowadays :)
This domain is owned by us.